server/rooms.ts

```diff
@@ -679,6 +692,11 @@ export abstract class BasicRoom {
 	 */
 	async rename(newTitle: string, newID?: RoomID, noAlias?: boolean) {
 		if (!newID) newID = toID(newTitle) as RoomID;
+		try {
+			this.validateTitle(newTitle, newID);
+		} catch (e) {
+			return;
```

Comment:
?

Reply:
Oh, this is because of the async `Chat.ErrorMessage` problem. That's fixed now, so you can just make this make sense again.